src/main/java/homeworks/Infrastructure/ConfiguratiomManager.java

```diff
@@ -0,0 +1,33 @@
+package homeworks.Infrastructure;
+
+public class ConfiguratiomManager {
```

Comment:
rename to ConfigurationManager, my mistake on lesson ))

Reply:
done